test/workerd/tests.mjs

```diff
@@ -102,3 +102,18 @@ export const util_implements = {
     assert.strictEqual(types.isAnyArrayBuffer(new ArrayBuffer(0)), true);
   },
 };
+
+// --- node:path
+
+export const path_implements = {
```

Comment:
I think was I tested and was not working was path and directory separators

Reply:
@jasnell's pull-request addresses this https://github.com/cloudflare/workerd/pull/3015

Reply:
Changes required:
- pathModule.~default~
- Update `workerd`

-> we won't be able to merge this until workerd is updated. Could you please split this PR in 2 different PRs so that we can revert the `sys` change ASAP.